client/src/utils/displayGenerator.js

```diff
@@ -57,7 +65,11 @@ export function renderDisplayFields(fields, exclude = null) {
             <Typography gutterBottom variant="subheading" color="textSecondary">
               {label}
             </Typography>
-              {paragraphs}
+            <div style={fontStyle}>
```

Comment:
I have wild idea (not guaranteed to work) to avoid manually styling the `div` to match `Typography` :

```jsx
<Typography component='div'>
  <ReactMarkdown>
    {value}
  </ReactMarkdown>
</Typography>
```